pkgs/applications/graphics/rapid-photo-downloader/default.nix

```diff
@@ -54,11 +61,14 @@ python3Packages.buildPythonApplication rec {
     requests
     colorlog
     pyprind
-  ]);
+  ];
 
   makeWrapperArgs = [
     "--set GI_TYPELIB_PATH \"$GI_TYPELIB_PATH\""
     "--set PYTHONPATH \"$PYTHONPATH\""
+    "--prefix PATH : ${stdenv.lib.makeBinPath [ exiftool ]}"
+    "--prefix LD_LIBRARY_PATH : ${stdenv.lib.makeLibraryPath [ libmediainfo ]}"
+    "--prefix GST_PLUGIN_SYSTEM_PATH_1_0 : \"$GST_PLUGIN_SYSTEM_PATH_1_0\""
```

Comment:
Why is this needed? Shouldn't this be `--set`?

Reply:
Without setting `GST_PLUGIN_SYSTEM_PATH_1_0`, gstreamer can't find its plugins, and thus fails to generate thumbnails for video files.

`--set` seems reasonable to me, but virtually every other occurrence of `GST_PLUGIN_SYSTEM_PATH` in nixpkgs, including `wrapGAppsHook`, uses `--prefix`. Out of more than two dozen occurrences, only one (beets) uses `--set`.

Reply:
Okay, let's keep `--prefix`.